### PR TITLE
Chef 15 fixes

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

**Comment by ramereth** [comment]
[Chef 15: osl-php](https://trello.com/c/nqmS7ld3/1730-chef-15-osl-php)

File: test/cookbooks/php_test/recipes/phpinfo_apache.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 major_version = node['php']['version'].to_i
-node.normal['apache']['mod_php']['module_name'] = "php#{major_version}"
-node.normal['apache']['mod_php']['so_filename'] = "libphp#{major_version}.so"
+node.default['apache']['mod_php']['module_name'] = "php#{major_version}"
+node.default['apache']['mod_php']['so_filename'] = "libphp#{major_version}.so"
 
 include_recipe 'apache2'
```
Comment (detjensrobert) [inline test/cookbooks/php_test/recipes/phpinfo_apache.rb:5]: This recipe still uses the old apache2 cb resources.

**Comment by ramereth** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v5.2.0.  Have a nice day!

**Pull request merged**